### ISSUE TITLE
Cloning private repositories

### ISSUE THREAD

**Issue by tombouctou** [creationix/jsgit#4]
Hi,
I don't know if ssh auth development is planned, but now I receive the following error:

```
jsgit clone git://git@github.com:crowdlab/crowdlab.git
Cloning repo into 'crowdlab'...

/usr/local/lib/node_modules/js-git-node/node_modules/git-fetch/fetch.js:31
      if (caps["side-band-64k"]) {
```

trying to clone my private repo.

**Comment by creationix** [comment]
Yes, but this repo is temporarily broken while I refactor the core library.

Just today I got ls-remotes working over https with auth and ssh using password or private keys. https://gist.github.com/creationix/6178532#file-ssh-log

As I implement clone again, it should work in all three transport styles.  Then I'll rewrite this library and have a nice solid CLI git tool written in node.
